### ISSUE TITLE
Share channels list is incomplete for staff users

### ISSUE THREAD

**Issue by karina4now** [TRASH-INC/trash-inc.github.io#200]
When a user creates a new post, on the client in VideoUploader.swift, `MyAvailableChannelsQuery` (`myAvailableChannels`) is sent to the back end to get a list of the possible channels that the user can post to.

If the current user of the app has the staff role, then after the user taps the director's Share to TRASH button, the user is presented to choose one of these channels.

The client expects that the `myAvailableChannels` query will return multiple channels. However, as of mid-February 2020, the back end is only returning a single special channel whose `name` field is `PROFILE`. If this channel is selected, the video is supposedly posted to the user's profile screen only, and not to an actual channel. It's unclear if this actually works.

The staff user is also given the option of selecting "Default", in which case the back end chooses which channel to post the video to, based on its own logic.

The bug is that since only the `PROFILE` channel is returned by the back end, and not any real channels, the experience is confusing.

To work around this issue, on Feb 18, 2020, the client was modified to skip presenting the action sheet (see below) in this case, and will behave as if Default is selected. If the server is ever modified to have `MyAvailableChannelsQuery` return multiple channels, then the client will correctly display them.

**TL;DR:** If we want to give staff users the option of choosing which channel to post their videos to, the back end should return multiple channels for the `myAvailableChannels` query instead of just the channel whose `name` field is `PROFILE`.

![Simulator Screen Shot - iPhone 11 - 2020-02-18 at 14 22 20](https://user-images.githubusercontent.com/12141562/74786255-1a9ca400-5261-11ea-9389-652219e98dba.png)
